C8_low_latency.md

```diff
@@ -0,0 +1,178 @@
+# More info: Low latency kernel
+
+On the desktop used to control the real robot, it is adviced (but not mandatory) to use a low latency kernel.
+
+## How to evaluate the latency of the kernel
+
+Clone pam_demos:
+
+```
+git clone https://github.com/intelligent-soft-robots/pam_demos.git
+```
+
+In a first terminal, start a dummy robot, using the default configuration:
+
+```
+o80_dummy
+```
+
+start the frequency check demos:
+
+```
+cd /path/to/pam_demos/frequency_check
+python ./frequency.py
+```
+
+This will display some stats regarding the frequency at which the backend
+is running. Here an example of results obtained using a regular kernel
+(results may grandly vary depending on the hardware and kernel version):
+
+```
+Frequency monitoring, over 10001 iterations
+	expected frequency: 500.0
+	spikes over 525.0: 45 (0.44995500449955006%)
+	spikes over 550.0: 20 (0.1999800019998%)
+	spikes below 475.0: 40 (0.3999600039996%)
+	spikes below 450.0: 19 (0.18998100189981001%)
+	max frequency observed: 35625.22265764161
+	min frequency observed: 209.46764426612697
+	average frequency (spikes excluded): 500.1007930571098
+	standard deviation (spikes excluded): 7.307281438308859
+```
+
+You can also visually observe the variations in frequency by using ```o80_plotting```
+(simply call ```o80_plotting``` in a terminal).
+
+
+## How to install the low latency kernel
+
+### install the kernel
+
+In a terminal:
+
+```
+sudo apt install linux-lowlatency
```

Comment:
Just a side note: There is also `linux-lowlatency-hwe-20.04` (adjust numbers based on Ubuntu version that is used), which seems to provide a newer version of the kernel.  But I think the exact version is not really relevant in most of the cases, so probably both are fine.